### DISCUSSION TITLE
Dialog content not accept HTML tags

### DISCUSSION THREAD

**Discussion by marcosgomesneto** [tusen-ai/naive-ui#1352] [General]
I'm trying to insert HTML code into the content of the Dialog component. But the code doesn't work, Dialog converts HTML into plain text.

I want to insert a line break, but it doesn't accept "<br>" and it doesn't accept "\r\n" either

**Comment by 07akioni** [comment]
> I'm trying to insert HTML code into the content of the Dialog component. But the code doesn't work, Dialog converts HTML into plain text.
> 
> I want to insert a line break, but it doesn't accept "" and it doesn't accept "\r\n" either

It's not recommend to use raw HTML, It's better to use render function.

If you do require it I think we can add a `contentHtml` option (but still not recommended).

Please share your opinion on it.

**Comment by marcosgomesneto** [comment]
Thank You, I used the render and it worked. No code changes needed.

Congratulations on your framework, I'm really enjoying it.

Att,
Marcos G.

**Reply by clabnet** [reply]
@marcosgomesneto 
Sorry, I'm new to Naive-Ui. Could you please explain me how to render a string html (including `<br/>`) into content ? Thank's

**Reply by clabnet** [reply]
I found the solution. 
```
 content: () => {
        return h('div', {
          class: 'text-red-600',
          innerHTML:
            'This operation and subsequent ones will be reverted and the tree will be restored to previous state.<br/><br/><b>Are you sure you want to proceed ?</b>'
        })
      }
```
